### ISSUE TITLE
Upcoming Trainings

### ISSUE THREAD

**Issue by adamhenriques** [Praqma/praqma.github.io#157]
Can we somehow get an overview of all upcoming trainings or get dates on the courses sheduled?

**Comment by meekrosoft** [comment]
Yes, the training landing page should have a list of the upcoming courses.

**Comment by monaslarsen** [comment]
I will sketch a design that both makes the training types easier to overview AND gives a pretty calendar/list etc for the scheduled events
